src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/Poll.java

```diff
@@ -0,0 +1,127 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class Poll extends BasePageObject {
+
+    @Getter
+    @FindBy(css = ".poll-title__input")
+    private WebElement titleInput;
+
+    @Getter
+    @FindBy(css = ".poll-title")
+    private WebElement title;
+
+    @Getter
+    @FindBy(css = ".poll-footer__vote")
+    private WebElement voteButton;
+
+    @FindBy(css = ".poll-footer__voted")
+    private WebElement votedMessage;
+
+    @Getter
+    @FindBy(css = ".poll-answer__input")
+    private List<WebElement> answersInputsList;
+
+    @Getter
+    @FindBy(css = ".poll-answer")
+    private List<WebElement> answersList;
+
+    @Getter
+    @FindBy(css = ".poll-answer__radio")
+    private List<WebElement> answersRadioButtonsList;
+
+    @Getter
+    @FindBy(css = ".poll-answer__result-bar.is-active")
+    private WebElement activeBarResult;
+
+    @Getter
+    @FindBy(css = ".poll-answer__result-bar")
+    private List<WebElement> BarResultsList;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "BarResultsList" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)